components/EpisodeListItem/index.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledListItem = styled.li`
+  display: flex;
+  flex-direction: row;
```

Comment:
With `flex-direction: row` you set the default value. To do that is not necessary.

Reply:
changed it